docs/Change_Log.md

```diff
@@ -1,5 +1,11 @@
 # Change Log
 
+#### April 3, 2023
+
+## Interaction Channel Data
+
+Interactions now contain a `channel` field which is a partial channel object and guaranteed to contain `id` and `type`. We recommend that you begin using this channel field to identify the source channel of the interaction, and may deprecate the existing `channel_id` field in the future. See the [interaction documentation](#DOCS_INTERACTIONS_RECEIVING_AND_RESPONDING/interaction-object) for more details.
```

Comment:
>  guaranteed to contain `id` and `type`.
Can't this be removed because the `Channel` object already has these fields as non-nullable and non-optional?